src/api/Feed.js

```diff
@@ -622,29 +617,54 @@ class Feed extends Base {
             }
         };
 
+        const v2Promises = [
+            versionsPromise,
+            currentVersionPromise,
+            commentsPromise(),
+            tasksPromise,
+            appActivityPromise,
+            annotationsPromise,
+        ];
+
+        const fetchV2FeedItems = async promises => {
+            return Promise.all(promises).then(
+                ([versions: ?FileVersions, currentVersion: ?BoxItemVersion, ...feedItems]) => {
+                    const versionsWithCurrent = currentVersion
+                        ? this.versionsAPI.addCurrentVersion(currentVersion, versions, this.file)
+                        : undefined;
+                    return sortFeedItems(versionsWithCurrent, ...feedItems);
+                },
+            );
+        };
+
+        const compareV2AndUaaFeedItems = async (uaaFeedItems, uaaResponse) => {
```

Comment:
i don't think you need `async` in either function since `await` is not being used